learning_journal/learning_journal/views.py

```diff
@@ -0,0 +1,95 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+import transaction
+import datetime
+from jinja2 import Markup
+import markdown
+from sqlalchemy import update
+from .forms import EntryCreateForm, EntryUpdateForm
+from pyramid.httpexceptions import HTTPFound
+
+
+from sqlalchemy.exc import DBAPIError, ResourceClosedError, IntegrityError
+
+from .models import (
+    DBSession,
+    Entry,
+)
+
+
+# @view_config(route_name='list', renderer='string')
+@view_config(route_name='list', renderer='templates/pretty.jinja2')
+def list_view(request):
+    display = DBSession().query(Entry.metadata.tables['entries']).all()[::-1][:10]
```

Comment:
You'll only ever see 10 entries here, eh?

And what is the order?  Not correct.  You want to order them by the date they were created.  This view will not do that.